test/controllers/update/update.review.statement.controller.spec.ts

```diff
@@ -95,7 +95,7 @@ describe("Update review overseas entity information controller tests", () => {
       expect(resp.text).toContain(UPDATE_CHECK_YOUR_ANSWERS_CONTACT_DETAILS);
       expect(resp.text).toContain(NO_CHANGE_REVIEW_STATEMENT_WHO_CAN_WE_CONTACT);
       expect(resp.text).toContain(CONTINUE_BUTTON_TEXT);
-      expect(mockIsActiveFeature).toHaveBeenCalledTimes(4);
+      expect(mockIsActiveFeature).toHaveBeenCalledTimes(5);
```

Comment:
We could also confirm `getBeneficialOwnersPrivateData` has been called too

Reply:
line 101